### PR TITLE
Update appengine-web.xsd with new staging elements

### LINKED ISSUE #2862: appengine-web.xml validation doesn't work with staging elements
The new staging elements are available for `appengine-web.xml`.

https://cloud.google.com/appengine/docs/standard/java/config/appref#staging_elements

However, XML validation doesn't work and places an error. We don't allow deploying if there is an error.

selection_001

A workaround to force deploying is to temporarily delete the problem marker:

selection_003

### PR DESCRIPTION
**Pull request by chanseokoh**
Fixes #2862. I've retrieved `appengine-web.xsd` from the most recent Cloud SDK.

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/project/ProjectUtils.java [hunk 1]
```diff
@@ -149,26 +149,30 @@
...
 
-  public static void waitUntilNoBuildErrors() throws CoreException {
+  public static boolean waitUntilNoBuildErrors() throws CoreException {
     IProject[] projects = getWorkspace().getRoot().getProjects();
-    waitUntilNoBuildErrors(projects);
+    return waitUntilNoBuildErrors(projects);
   }
 
-  public static void waitUntilNoBuildErrors(IProject... projects) throws CoreException {
+  public static boolean waitUntilNoBuildErrors(IProject... projects) throws CoreException {
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/project/ProjectUtils.java]: Could you add some javadoc for what the return means?

**Comment by chanseokoh** [comment]
This affects tests on a different test class. I wonder if the new XSD isn't compatible with them.

**Comment by chanseokoh** [comment]
This is very weird. The tests also fail on master when running inside Eclipse.

File: plugins/com.google.cloud.tools.eclipse.appengine.validation/xsd/appengine-web.xsd [hunk 2]
```diff
@@ -66,6 +66,8 @@
       <xs:element type="ns:beta-settingsType" name="beta-settings" minOccurs="0" xmlns:ns="http://appengine.google.com/ns/1.0"/>
       <xs:element type="ns:vm-health-checkType" name="vm-health-check" minOccurs="0" xmlns:ns="http://appengine.google.com/ns/1.0"/>
       <xs:element type="ns:vm-health-checkType" name="health-check" minOccurs="0" xmlns:ns="http://appengine.google.com/ns/1.0"/>
+      <xs:element type="ns:vm-liveness-checkType" name="liveness-check" minOccurs="0" xmlns:ns="http://appengine.google.com/ns/1.0"/>
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.validation/xsd/appengine-web.xsd]: It's weird that the ns prefix is getting declared on each element.

**Comment by chanseokoh** [comment]
I remember Abbie said some tests run OK with `mvn verify` but fail inside Eclipse. Looks like this is happening on master.

**Comment by chanseokoh** [comment]
The previous test failures were due to #2868.

**Comment by chanseokoh** [comment]
Tests passed. Merging

**Pull request merged**